### DISCUSSION TITLE
Team Versus

### DISCUSSION THREAD

**Discussion by nwai90** [bafolts/terraforming-mars#1008] [Ideas]
For 4 players (2v2) or 6 players (2v2v2). Teammates must sit opposite each other (in 4p) or 2 spaces apart (in 6p).

Modifications:
- Total up members' scores for each team at game end
- Tiles may be placed adjacent to teammates' placements
- Milestones and awards are still fulfilled and scored separately, however 2 players on the same team can win both 1st and 2nd place for an award

**Comment by kevinb9n** [comment]
I like it! I hope to try it if I ever get a F2F 4p game again.

Possible alternative: team score is the smaller of the two player scores - so a player who is doing very well has to think a little more about the partner. OTOH, could result in too much counting up of VPs etc.

Cheating (in the form of back-channel communication with your partner) can't really be prevented.

**Comment by nwai90** [comment]
@kevinb9n Definitely true here. We didn't have that problem in real life games :)

**Comment by kevinb9n** [comment]
Apparently Curiosity does this:

```
Curiosity
BOT
Today at 2:42 AM
Rules for 2 v 2
1. Four players. You sit across from your teammate.
2. You can look at your teammate's cards at any time.
3. You can talk to your teammate privately at any time.
4. At end game, combine the score of your team and compare that to the other team's.
5. Unfriendly tie. You split points if tie for an award (3 pts if tie for first, 1 if tie for second).
6. Ban Poseidon and Philares.
```

Not sure where that comes from or what it's for. I would personally just not follow 2 and 3 as it just makes it basically one player controlling 2 seats. The addition of #5 would remove an award frontrunner's incentive to sandbag (when they feel sure their partner can tie them).

While I haven't tried it yet, I am very intrigued by your greeneries tweak. It really seems like it would make things interesting. I assume that tweak would also affect where Arcadian can place cubes.
